aries_vcx/tests/test_creds_proofs.rs

```diff
@@ -313,7 +312,7 @@ mod integration_tests {
     #[ignore]
     async fn test_agency_pool_generate_proof() {
         SetupProfile::run(|mut setup| async move {
-            create_and_store_credential(
+            let (_, _, _, _, _, _, _, _, _, _, tails_dir, _) = create_and_store_credential(
```

Comment:
Oh, good ol' functions returning 12 values in a tuple, always make me smile (and / or cry) :)

Reply:
yeah I know ^_^ 

A task for another day to sort these out